.gitignore

```diff
@@ -1,3 +1,4 @@
+.idea/
```

Comment:
@tolry We're actually versioning the `.idea` folder in order to share IntelliJ settings across developers (https://intellij-support.jetbrains.com/hc/en-us/articles/206544839-How-to-manage-projects-under-Version-Control-Systems) so I can't have it in the global gitignore. However, I will remove the `.gitignore` change from this PR.